codegen/templates/endpoint_register.tmpl

```diff
@@ -88,4 +91,36 @@ func Register(g *zanzibar.Gateway, router *zanzibar.Router) {
 		),
 	)
 	{{end}}
+
+	// TODO: (lu) remove below, to be generated
```

Comment:
We cannot put this here as it breaks edge-gateway. We need to use custom endpoints like contacts for this.